src/commands/deleteCommand.ts

```diff
@@ -0,0 +1,21 @@
+import { Command } from "@yuudachi/framework";
+import type { InteractionParam, ArgsParam, CommandMethod } from "@yuudachi/framework/types";
+import { PermissionFlagsBits } from "discord.js";
+import type { DeleteCommandContextCommand } from "../interactions/context/deleteCommand.js";
+
+export default class extends Command<typeof DeleteCommandContextCommand {
+	public constructor() {
+		super(["Delete Command"]);
+	}
+	public override async messageContext(
+		interaction: InteractionParam<CommandMethod.MessageContext>,
+		args: ArgsParam<typeof DeleteCommandContextCommand>,
+	): Promise<void> {
+	   if(args.message.interaction.user.id !== interaction.user.id || !interaction.member.permissions.has(PermissionFlagsBits.ManageMessages)){
```

Comment:
The linter is not going to be happy with your changes I'm afraid
```suggestion
	   if (args.message.interaction.user.id !== interaction.user.id || !interaction.memberPermissions.has(PermissionFlagsBits.ManageMessages)) {
```